tsfc/loopy.py

```diff
@@ -264,9 +264,6 @@ def generate(impero_c, args, scalar_type, kernel_name="loopy_kernel", index_name
         preambles=preamble
     )
 
-    # Prevent loopy interchange by loopy
-    knl = lp.prioritize_loops(knl, ",".join(ctx.index_extent.keys()))
```

Comment:
This could have some performance degradation. For example, for kernels of the form:
```python
u[n,j] = 1.
```
it is preferred to have "j"-loop nested within "n"-loop. This preferred nesting can be guaranteed by performing appropriate calls to `lp.prioritize_loops`. Without those priorities loopy might generated code with the loops permuted in an unfavorable manner.

Reply:
Ah good to know! I should dig into how we build this `ctx.index_extent` because it is including 'sibling' loop nests when it shouldn't.

Reply:
#333 has a 'proper' solution to this. Thanks for flagging it.